content/about/employment.rst

```diff
@@ -7,6 +7,19 @@ Open Source Lab Employment
 Employment
 ----------
 
-*Here are the current student positions we have available for hire. All
-positions require you to pass a security background check if selected for
-employment.*
+The Open Source Lab hires undergraduates for an on-campus, paid internship, 20
+hours a week year-round, typically for two or more years, allowing students to
+continue taking classes while gaining real-world experience.
+
+* Students participate in all phases of a project, including application design,
+  development, testing, delivery, hosting, and support of industrial strength
+  applications.
+* Professional staff provides ongoing mentoring in software development,
+  testing, and system administration best practices.
+* Students interact directly with clients to experience the business side of
+  software engineering and application hosting.
+
+**Positions**
+
+* Student Developer
+* Student Systems Engineer
```

Comment:
Oh okay! If it weren't Friday I probably could have guessed that. I'll be sure to add that in.